src/Microsoft.FeatureManagement/Telemetry/EvaluationEvent.cs

```diff
@@ -18,7 +18,7 @@ public class EvaluationEvent
         /// <summary>
         /// The targeting context used to evaluate the feature.
         /// </summary>
-        public TargetingContext TargetingContext { get; set; }
+        public ITargetingContext TargetingContext { get; set; }
```

Comment:
I'd like to keep this as `TargetingContext` to enable easy expansion. We should be able to create a new instance and copy over in FeatureManager.